src/components/styles.module.css

```
@@ -0,0 +1,24 @@
+.todolist {
+    margin: 30px auto 0;
+    text-align: center;
+    font-family: 'Raleway', sans-serif;
+}
+
+.groupTask {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.todolist__list {
```

Мы не используем БЭМ. Блок (в терминологии БЭМ) нам добавляют css модули когда к классам добавляют путь файла
```suggestion
.list {
```